### ISSUE TITLE
Write batch script to set alt text for images to title if not already set

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-a11y#2]
For oik-plugins ( s.b/oikcom ) I received quite a few messages from WAVE for "Missing alt text".
For quite a few of these I needed to set the alt text for the image.
To save time I asked ChatGPT for a WP-CLI snippet to set the alt text for images from the title if the alt text was not already set.
This is OK if you have access to WP-CLI on your hosting.
But it could also be written as a "batch" routine to run from the wp-admin.

**Comment by bobbingwide** [comment]
Command to run the script from WP-cli
Change directory to the oik-a11y plugin folder
```
wp eval-file set-alt-from-title.php 
```
